hash/nes.xml

```diff
@@ -3388,6 +3430,33 @@ license:CC0
 		</part>
 	</software>
 
+<!-- This is on a Famicom PCB, but it already contains the US code... -->
+	<software name="batmanrjup" cloneof="batmanrj">
+		<description>Batman - Return of the Joker (USA, Prototype)</description>
```

Comment:
Please don’t capitalise things that are not proper nouns or initialisms in description text.